src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -160,6 +140,24 @@ private static void setupImporters() {
         ImporterFactory.addImporter("json", new JsonCommandImporter());
     }
 
+    private static void setupFeaturesMenu(Application application) {
+        Job2dDriver loggerDriver = new LoggerDriver(false);
+        AppFeature.addFeature("Simple Logger", loggerDriver);
+
+        Job2dDriver loggerDriver2 = new LoggerDriver(true);
+        AppFeature.addFeature("Detailed Logger", loggerDriver2);
+
+        //overflow
+        Job2dDriver recordingDriver = new RecordingDriverDecorator(DriverFeature.getDriverManager().getCurrentDriver());
+        AppFeature.addFeature("Recording Support", recordingDriver);
+
+        //overflow
+        Job2dDriver usageMonitorDriver = new UsageMonitorDriverDecorator(DriverFeature.getDriverManager().getCurrentDriver());
+        AppFeature.addFeature("Usage monitor", usageMonitorDriver);
```

Comment:
I think, that decorators should be handled differently. Recording and UsageMonitor features can be devoid of decorator pattern and it should help in those cases. 
 
You should also consider Transforming features (see PR #48).